azurerm/resource_arm_app_service.go

```diff
@@ -82,6 +82,12 @@ func resourceArmAppService() *schema.Resource {
 				Default:  false,
 			},
 
+			"client_cert_enabled": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  false,
```

Comment:
Could we remove this default here so we only populate the request if something is specified?